onnxruntime/contrib_ops/cpu/attnlstm/deep_cpu_attn_lstm.h

```diff
@@ -96,11 +96,7 @@ class DeepCpuAttnLstmOp final : public OpKernel {
 // across them. mutable due to this.
 // The alternative would be to create a threadpool in each call to Compute but that would incur thread creation
 // cost on every call.
-#ifdef USE_EIGEN_THREADPOOL
-  mutable Eigen::NonBlockingThreadPool ttp_{static_cast<int>(std::thread::hardware_concurrency())};
-#else
-  mutable TaskThreadPool ttp_{std::thread::hardware_concurrency()};
-#endif
+  mutable onnxruntime::concurrency::ThreadPool ttp_{"DEEPCPU_ATTN_LSTM", std::thread::hardware_concurrency()};
```

Comment:
>nnxruntime::concurrency::ThreadPoo [](start = 11, length = 34)

So we still intend to keep its own threadpool for LTSM? Would this not defeat the purpose?

Reply:
For this one, I think we can do it in another pr. I already have one for GRU.

Reply:
Correct. This is an intermediate stage.  I need to modify LSTM, GRU, etc... to use the operator threadpool instead of its own.  That is in my next PR.